integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -101,6 +102,19 @@ public static ObjectNode createMetadata(String title) {
     return obj;
   }
 
+  public static Map<String, Object> createMetadata(String title) {
```

Comment:
I don't really like having two versions of this function, but I think that because we have different input types for upsert and update, we need two versions now.